subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/FasterIncrementalAndroidBuildsPerformanceTest.groovy

```diff
@@ -68,6 +68,44 @@ class FasterIncrementalAndroidBuildsPerformanceTest extends AbstractCrossBuildPe
         results
     }
 
+    def "file system watching baseline non-abi change (build comparison)"() {
+        given:
+        runner.buildSpec {
+            displayName("with file system watching")
+            configureForNonParallel(delegate)
+            testProject.configureForNonAbiChange(delegate)
+            invocation {
+                args.addAll(Optimization.WATCH_FS.arguments)
+            }
+        }
+        runner.buildSpec {
+            displayName("with file system watching - Gradle 6.7")
```

Comment:
I wonder if we need to add the version number here. Gradle Profiler should add all the distinguishing data. WDYT?

Reply:
How would you call this version? `with file system watching - baseline`?

Reply:
That's good, but we could also keep it with the same name.

In any case we should move it to be the first scenario I think.

Reply:
We can't have two buildSpecs with the same display name.

Reply:
Mhm. "Gradle 6.7" is fine then I guess.